### PR TITLE
Newsline

### PR DESCRIPTION
**Pull request by DariaPrudnikova**

### CODE REVIEW

File: task3/index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
+<!doctype html>
+
```
Comment (lexastef) [inline task3/index.html]: html tag is missing

File: task3/index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
+<!doctype html>
+
+<head>
+    <title>Gossip</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header_logo">
```
Comment (lexastef) [inline task3/index.html]: Cosmetic: class name notation, use kebab-case

File: task3/index.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+<head>
+    <title>Gossip</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header_logo">
+            <a><img src="./images/logo.png" alt="Logo"></a>
```
Comment (lexastef) [inline task3/index.html]: anchor tag have a completely different behavior w/o href attribute.

File: task3/index.html [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        </div>
+        <div class="header_search">
+            <img class="header_search_img" src="images/search.png">
```
Comment (lexastef) [inline task3/index.html]: It's better to close single tags:
```
<input ... />
<img ... />
<br/>
```

File: task3/index.html [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    <main class="main">
+        <div class="new-post">
+            <img class="new-post_avatar" src="images/user_avatar.png">
+            <input class="new-post_description" type="text" placeholder="What's happening?">
+            <img class="new-post_add-image" src="images/add_image.png">
+            <img class="new-post_create-post-button" src="images/create_post.png">
+        </div>
+        <div class="main_filter">
+            <p class="filter" type="text">Filter by...</p>
+            <input class="filter_search_name" type="search" placeholder="@name">
```
Comment (lexastef) [inline task3/index.html]: The right types and updated markup for filers

**Comment by DariaPrudnikova** [comment]
Check please

**Pull request merged**